config/config.exs

```diff
@@ -0,0 +1,13 @@
+use Mix.Config
+
+patterns = [
+  # Credit Cards
+  ~r/#?(?:4[0-9]{12}(?:[0-9]{3})?|5[1-5][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11})/,
```

Comment:
In those situations I'd rather see the developer to define their own patterns than us try to get too crazy.  I'm trying to think of a few sane defaults out-of-the-box for folks who have no interest in configuration but want to make sure some of the more obvious things are addressed.